sdk/core/core-rest-pipeline/src/policies/tracingPolicy.ts

```diff
@@ -130,7 +130,10 @@ function tryCreateSpan(request: PipelineRequest, userAgent?: string): Span | und
   }
 }
 
-function tryProcessError(span: Span, err: any): void {
+export function tryProcessError(err: any, span?: Span): void {
+  if (!span) {
```

Comment:
`tryTraceError` better? `Process` sounds like we will do something to the error.

Reply:
Ohh I like that! I’ll do that change! Thank you!